### PR TITLE
Resolved the broken URL in /widgets/overview for issue #6308

### LINKED ISSUE #6308: [docs]: Fix the broken URL in /widgets/overview
### Summary

The hyperlink `Event Handlers` in the section "[Components : Overview](https://docs.tooljet.com/docs/widgets/overview)" is broken

Screenshot 2023-05-09 at 4 32 27 PM

To fix this, replace the hyperlink with https://docs.tooljet.com/docs/widgets/overview/#component-event-handlers

### Issue Type

Documentation bug

### The entire URL of the documentation with the issue

https://docs.tooljet.com/docs/widgets/overview/

### Steps to reproduce the issue

_No response_

### Additional Information

_No response_

### If the issue is confirmed, would you be willing to submit a pull request?

No

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

### PR DESCRIPTION
**Pull request by GODINME**
Corrections made for issue #6308 on the docs overview.md file URL broken link for even handlers. 

fixes #6308

### CODE REVIEW

File: docs/docs/widgets/overview.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Overview
 
 # Components: Overview
 
-Components are used to build the UI of the applications. They can be dragged onto the canvas from the Component Library and can be modified from the Properties Panel without needing to write any code. **[Event Handlers](/docs/actions/event-handler)** in Components allow end users to trigger queries and other application events to perform the **[Actions](/docs/category/actions-reference)**.
+Components are used to build the UI of the applications. They can be dragged onto the canvas from the Component Library and can be modified from the Properties Panel without needing to write any code. **[Event Handlers]( https://docs.tooljet.com/docs/widgets/overview/#component-event-handlers)** in Components allow end users to trigger queries and other application events to perform the **[Actions](/docs/category/actions-reference)**.
```
Comment (tejakummarikuntla) [inline docs/docs/widgets/overview.md:8]: Hey @GODINME , Thanks for the PR.
It's suggested to follow the existing formatting in URLs,
Kinldy, replace the URL https://docs.tooljet.com/docs/widgets/overview/#component-event-handlers with `docs/widgets/overview#component-event-handlers`

**Review by tejakummarikuntla — CHANGES_REQUESTED** [review]

**Comment by withshubh** [comment]
Hey @GODINME - Please sign the CLA

File: docs/docs/widgets/overview.md (hunk 1 above)
Comment (GODINME) [reply docs/docs/widgets/overview.md:8]: Okay @tejakummarikuntla

Comment (GODINME) [reply docs/docs/widgets/overview.md:8]: @tejakummarikuntla  Just realized I made a space typo and have fixed it, Do you still want me to replace the link with your suggestion or push the verified correction made

Comment (withshubh) [inline docs/docs/widgets/overview.md:8]:
```suggestion
Components are used to build the UI of the applications. They can be dragged onto the canvas from the Component Library and can be modified from the Properties Panel without needing to write any code. **[Event Handlers](/docs/widgets/overview#component-event-handlers)** in Components allow end users to trigger queries and other application events to perform the **[Actions](/docs/category/actions-reference)**.
```

**Review by withshubh — APPROVED** [review]

**Review by Navaneeth-pk — APPROVED** [review]

**Pull request merged**